src/transformStateWithClones.js

```diff
@@ -7,7 +7,28 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const clone1 = { ...state };
```

Comment:
```suggestion
  const clone1 = { ...state };
```
we don't have clone2 anymore, so you can change name clone1 to clone ;)

Reply:
thanks for the helpful tips, i fixed it 👍